README.md

```
@@ -49,6 +49,64 @@ This theme collection is based on the excellent [Gruvbox Theme by jdinhify](http
+- npm (comes with Node.js)
+
+### Build Process
+
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/vatsalsy/gruvbox_custom_themes.git
+   cd gruvbox_custom_themes
+   ```
+
+2. Run the build script:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)